server/chat-commands.js

```diff
@@ -2561,7 +2561,6 @@ const commands = {
 		Ladders.cancelSearches(targetUser);
 		targetUser.resetName(true);
 		targetUser.send(`|nametaken||${user.name} considers your name inappropriate${(reason ? `: ${reason}` : ".")}`);
-		targetUser.trackRename = targetUser.name;
```

Comment:
wtf has this always been broken?

Reply:
It seems like the correct solution is to move this above `resetName`, though.

Reply:
Also, the monitor messages on Main are correct. Do we have a private override of `/fr` or something?

Reply:
Weirdly this doesn't work on a fresh clone of the PS code, but it does work just fine on the main server

Reply:
Also the reason I didn't just move it above resetName is because this prevents redundancy if the forceRename function will ever be used anywhere else, and it ensures that the variable only gets set if the rename is successful

Reply:
Ok so uhh. I fucked this up initially but since there is still some non-pushed running this bug was never exposed lol. Plus the namefilter works fine still and most of the FRs come from banned names.

Reply:
You're being confused. The `forceRename` function just renames a user without checking an assertion; it has nothing to do with the `/forcerename` command used by moderators.

Reply:
Notice that the `/forcerename` command doesn't call the `forceRename` function at all.

Reply:
I requested it to be moved there so any function ever calling this method would trigger the name to be tracked, which I think is reasonable is it not? I don't think it's ever called with isForceRenamed when we don't want to track it.

Reply:
It does end up calling the function
```
	resetName(isForceRenamed = false) {
		return this.forceRename('Guest ' + this.guestNum, false, isForceRenamed);
	}
```

Reply:
I checked to confirm yes in any case that flag is set we want to track the rename, so it makes sense to move all the setting of user.trackRename there.

Reply:
The main thing is, if we ever introduce anything in the future that uses that flag, I don't want it to set `trackRename`.

Reply:
I guess overall that seems obscure enough that I don't mind doing it this way.